quickinfo/quickinfo-library/src/main/java/org/silverpeas/components/quickinfo/model/News.java

```diff
@@ -390,4 +377,24 @@ public int getNumberOfAttachments() {
             I18NHelper.defaultLanguage);
     return attachments.size();
   }
+
+  protected News() {
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Move this constructor to comply with Java Code Conventions. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://integration.silverpeas.org/sonar/coding_rules#rule_key=squid%3AS1213)